test/asserts/big-number-assert_test.js

```diff
@@ -20,31 +20,71 @@ const Assert = BaseAssert.extend({
  */
 
 describe('BigNumberAssert', () => {
-  it('should throw an error if the input value is not a big number', () => {
-    const choices = [[], {}, ''];
+  [undefined, { validateSignificantDigits: true }, { validateSignificantDigits: false }].forEach(option => {
+    describe(`with option '${option ? `{ validateSignificantDigits: ${option.validateSignificantDigits} }` : undefined }'`, () => {
```

Comment:
This can be simplified to:

```js
describe(`with option '${option}'`, () => {
```

Reply:
It works for `undefined` but not for the others:
> with option '[object Object]'

Reply:
You're right, I was too focused on that `undefined` 🙈